backend/src/test/kotlin/org/loculus/backend/controller/submission/DeleteSequencesEndpointTest.kt

```diff
@@ -317,8 +327,7 @@ class DeleteSequencesEndpointTest(
             .andExpect(status().isOk)
             .andExpect(content().contentType(MediaType.APPLICATION_JSON_VALUE))
             .andExpect(jsonPath("\$.length()").value(accessionVersions.size))
-            .andExpect(jsonPath("\$[0].accession").value(accessionVersions.first().accession))
-            .andExpect(jsonPath("\$[0].version").value(accessionVersions.first().version))
+            .andExpect(jsonPath("\$[*]", containsInAnyOrder(*expectedPairs.toTypedArray())))
```

Comment:
```suggestion
            .andExpect(jsonPath("\$[*]", containsInAnyOrder(expectedPairs)))
```

Reply:
Have you tested it? I'll commit it just for fun as revert is easy

Reply:
nice it worked!